web/vibe/web/web.d

```diff
@@ -183,7 +183,14 @@ URLRouter registerWebInterface(C : Object, MethodStyle method_style = MethodStyl
 		/*static if (isInstanceOf!(SessionVar, __traits(getMember, instance, M))) {
 			__traits(getMember, instance, M).m_getContext = toDelegate({ return s_requestContext; });
 		}*/
-		static if (!is(typeof(__traits(getMember, Object, M)))) { // exclude Object's default methods and field
+
+		// Ignore special members, such as ctor, dtors, postblit, and opAssign,
+		// and object default methods and fields.
+		// See https://github.com/vibe-d/vibe.d/issues/2438
+		static if (M != "__ctor" && M != "__dtor" && M != "__xdtor" &&
+				   M != "__postblit" && M != "__xdtor" && M != "__xpostblit" &&
```

Comment:
"__xdtor" is repeated twice. I guess there should be "__xctor"? 

Offtopic: does special members list exist somewhere out?

Reply:
> Offtopic: does special members list exist somewhere out?

good idea, druntime should define `isCompilerGeneratedMember(string)`, which would make this much more pleasant.

Reply:
Oooooor the language could be sane and not have magic names. Just a thought.

Reply:
Hm... another thought here, could we just do:

```D
static if (M.length >= 2 && M[0 .. 2] == "__")
```

Since double underscores are reserved for the language?

Reply:
does this come from an `__traits(allMembers)` ? If so it's already filtered.

Reply:
> does this come from an `__traits(allMembers)` ?

Yes.

> If so it's already filtered.

No, it's not.

Reply:
There's no such thing as `__xctor`, it was just a copy-paste error.
Reorganized things now and removed the duplication.

@schveiguy : We *could* in a future improvement, but that might affect code that shouldn't be affected, so I'd like to keep this a regression fix.

Reply:
> There's no such thing as __xctor, it was just a copy-paste error.

That's why I asked if there is the special members list. Thanks!

Reply:
I mean technically postblit isn’t ever going to show up for a class either. But I think it’s reasonable to include all magic functions in the list. It’s just that using double underscore checking would be future proof and a decent compromise.